pkg/apis/work/v1alpha1/work_types.go

```diff
@@ -82,6 +82,10 @@ type ManifestStatus struct {
 	// +kubebuilder:pruning:PreserveUnknownFields
 	// +optional
 	Status *runtime.RawExtension `json:"status,omitempty"`
+
+	// Health of current manifest, can be Healthy or Unhealthy.
+	// +optional
+	Health RefResourceHealth `json:"health,omitempty"`
```

Comment:
```suggestion
	// Health represents the healthy state of the current resource.
	// +kubebuilder:validation:Enum=Healthy;Unhealthy;Unknown
	// +optional
	Health ResourceHealth `json:"health,omitempty"`
```

Rename `RefResourceHealth` --> `Resourcehealth`